src/lib/viewers/media/MediaControls.js

```diff
@@ -877,6 +887,17 @@ class MediaControls extends EventEmitter {
         this.settings.addListener('subtitles', this.handleSubtitle);
         this.settings.loadSubtitles(subtitles, language);
     }
+
+    /**
+     * Takes a list of audio tracks and populates the settings menu
+     *
+     * @param {Array} audioLanguages - An ordered list of languages of the audio tracks
+     * @return {void}
+     */
+    initAlternateAudio(audioLanguages) {
+        this.settings.addListener('audiotracks', this.handleAudioTrack);
```

Comment:
No. Settings menu emits this whenever you choose an option. Shaka player doesn't seem to emit events when you manually choose a track. Why?